VoiceOfAmerica/ViewControllers/CandidateInitialCell.swift

```diff
@@ -36,4 +38,56 @@ final class CandidateInitialCell: UITableViewCell {
         return label
     }()
 
+    private let circleImage: UIImageView = {
+        var trump = UIImage(named: "donald-trump")
```

Comment:
Yes, we should add an image field to the Candidate object and pull it from there. I think I also want a Republican/Democrat boolean. This is a placeholder for now, because it's going to be a bit of a pain to import headshots for candidates.